### PR TITLE
Add init arguments in AddSelfLoops

### PR DESCRIPTION
**Pull request by dongkwan-kim**
This PR makes the AddSelfLoops transform can use full features of the add_self_loops method.

- choose the name of edge weights/features
- use fill_value in add_self_loops

### CODE REVIEW

File: torch_geometric/transforms/add_self_loops.py [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
 from torch_geometric.transforms import BaseTransform
 
 
 class AddSelfLoops(BaseTransform):
-    r"""Adds self-loops to the given homogeneous or heterogeneous graph."""
+    r"""Adds self-loops to the given homogeneous or heterogeneous graph.
+
+    Args:
+        attr: (str, optional): The name of the attribute of edge weights
+            or multi-dimensional edge features, to pass it to the
```
Comment (rusty1s) [inline torch_geometric/transforms/add_self_loops.py:15]:
```suggestion
            or multi-dimensional edge features to pass to
```

File: torch_geometric/transforms/add_self_loops.py [hunk 2]
```diff
@@ -1,22 +1,47 @@
...
+            :meth:`torch_geometric.utils.add_self_loops`.
+            (default: :obj:`edge_weight`)
```
Comment (rusty1s) [inline torch_geometric/transforms/add_self_loops.py:17]:
```suggestion
            (default: :obj:`"edge_weight"`)
```

File: torch_geometric/transforms/add_self_loops.py [hunk 3]
```diff
@@ -1,22 +1,47 @@
...
+        fill_value (float or Tensor or str, optional): The way to generate
+            edge features of self-loops (in case :obj:`edge_attr != None`).
```
Comment (rusty1s) [inline torch_geometric/transforms/add_self_loops.py:19]:
```suggestion
            edge features of self-loops (in case :obj:`attr != None`).
```

**Review by rusty1s — APPROVED** [review]
This is great. I only have some minor suggestions :)

**Comment by dongkwan-kim** [comment]
This looks really great. Thank you! 😄

**Comment by rusty1s** [comment]
I need to thank you :)

**Pull request merged**